traffic_ops/app/bin/tests/Dockerfile-golangtest

```diff
@@ -18,8 +18,13 @@ ARG DIR=github.com/apache/trafficcontrol
 ADD traffic_ops /go/src/$DIR/traffic_ops
 ADD lib /go/src/$DIR/lib
 
+COPY traffic_monitor/tests/gocover.bash /go/src/$DIR/traffic_ops/traffic_ops_golang/gocover.bash
+RUN chmod +x /go/src/$DIR/traffic_ops/traffic_ops_golang/gocover.bash
 WORKDIR /go/src/$DIR/traffic_ops/traffic_ops_golang
+RUN bash -c "go get -u github.com/jstemmer/go-junit-report"
+RUN bash -c "go get -v"
```

Comment:
You're not using any extended features of `bash` here, so there's no reason to force the fork to a shell that may not be in the `PATH` if the image's base is changed (unlikely though that is). You can do all of this in one step after the `WORKDIR` directive to reduce caching as well:

```Dockerfile
WORKDIR /go/src/$DIR/traffic_ops/traffic_ops_golang
RUN chmod a+x gocover.bash && \
    go get -u github.com/jstemmer/go-junit-report && \
    go get -v
```